.github/workflows/build-and-push-nosql-sec-image.yml

```diff
@@ -0,0 +1,52 @@
+name: Build and publish NoSQL container image to GitHub Container Registry
+
+on:
+  push:
+    branches:
+      - main
+    paths:
+      - 'NoSQL/ce-sec/*'
+      - '.github/workflows/build-and-push-nosql-image.yml'
+  workflow_dispatch:
+
+env:
+  IMAGE_NAME: nosql
+  TAG: ce
```

Comment:
These are the same `IMAGE_NAME` and `TAG` values as the existing NoSQL workflow. Please edit the existing file instead of creating a new one (or delete the old workflow).

Reply:
I am rewriting the tag later in the code - I need to review and delete the TAG from here. 

There is probably a misunderstanding here: there is no old workflow and a new one - we want to maintain two versions